packages/L10n/build-translations.js

```diff
@@ -13,7 +13,9 @@ const removeTranslationFiles = () => {
     }
 
     files.forEach(file => {
-      fs.unlinkSync(`${translationsPath}/${file}`);
+      if (file !== ".gitkeep") {
```

Comment:
we need to keep .gitkeep on the localization folder in order to version the folder in git and use it to process the locale generator script.